:book: docs enhancements (#77)
kairos-io/kairos-docs: content/en/docs/Reference/build-from-scratch.md

## Prerequisites

To follow the steps below, you'll need to have Docker or a container engine installed on your local machine. Additionally, note that the steps have been tested on Linux but should also work in other environments. 

The base OS used as an example in this document is Fedora 38. However, the steps should work with any other OS as well. The OS should meet the following requirements:

> - Use `systemd` (`openRC` is currently not supported, but in the roadmap)

Reviewer: maybe worth a link to the roadmap? not sure if we have a card for the pure alpine

Author: link added :+1:

- Use `grub` as bootloader

> - If the system supposedly is meant to be used with EFI, the kernel needs to have enabled the `CONFIG_EFI_STUB` option ( see: https://docs.kernel.org/admin-guide/efi-stub.html)

Reviewer: ```suggestion
- If the system will use EFI, the kernel needs to have enabled the `CONFIG_EFI_STUB` option ( see: https://docs.kernel.org/admin-guide/efi-stub.html)
```
